### PR TITLE
[Azure Stack] Added override of default long running operation

### PR DESCRIPTION
**Pull request by deathly809**
## Description

This SDK has operations which might result in the RP service going down.  We have had customer ask to increase the timeout on long running operations.

Newest spec PR: https://github.com/Azure/azure-rest-api-specs/pull/3791

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/AzureStack/Admin/FabricAdmin/Fabric.Admin/src/FabricAdminClient.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    /// <summary>
+    /// Fabric Admin Client
+    /// </summary>
+    public partial class FabricAdminClient : ServiceClient<FabricAdminClient>, IFabricAdminClient, IAzureClient
+    {
+
+        private int Minute = 60;
+
+        partial void CustomInitialize()
```
[the remark this answers is not in the document]
Comment (deathly809) [reply src/AzureStack/Admin/FabricAdmin/Fabric.Admin/src/FabricAdminClient.cs]: Done.

File: src/AzureStack/Admin/FabricAdmin/Fabric.Admin/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -16,4 +16,4 @@
 [assembly: NeutralResourcesLanguage("en")]
 
 [assembly: AssemblyVersion("0.2.0.0")]
-[assembly: AssemblyFileVersion("0.2.0.0")]
+[assembly: AssemblyFileVersion("0.2.1.0")]
```
[the remark this answers is not in the document]
Comment (deathly809) [reply src/AzureStack/Admin/FabricAdmin/Fabric.Admin/Properties/AssemblyInfo.cs]: Done.

**Comment by deathly809** [comment]
@shahabhijeet For some reason there is a _metadata folder under the AzureStack folder instead of SDKs folder, is this intended?  I have not touched that code path.

File: src/AzureStack/Admin/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj [hunk 3]
```diff
@@ -7,19 +7,12 @@
     <PropertyGroup>
         <PackageId>Microsoft.AzureStack.Management.Fabric.Admin</PackageId>
         <Description>Provides developers with libraries for the fabric admin under AzureStack to manage fabric infrastructure.</Description>
-        <VersionPrefix>0.2.0-preview</VersionPrefix>
+        <VersionPrefix>0.2.1-preview</VersionPrefix>
```
Comment (shahabhijeet) [inline src/AzureStack/Admin/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj]: Similar version should be reflected for your nuget package version.
Make it 0.3.0 as well

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
@deathly809 please add link to the spec change as well.
The current commit in SdkInfo*.cs file is pointing to a different commit and hence would like to make sure the meta data is insync with the generation.
Point the spec PR in every SDK generation PR.
Very soon, this will be automated, but until then please provide this information.

File: src/AzureStack/Admin/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj (hunk 3 above)
Comment (deathly809) [reply src/AzureStack/Admin/FabricAdmin/Fabric.Admin/Microsoft.AzureStack.Management.Fabric.Admin.csproj]: Good catch.

**Comment by deathly809** [comment]
@shahabhijeet I added the last PR made, this will be a breaking change, but the team was not ARM compliant.  I updated the version as well.

File: src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj [hunk 4]
```diff
@@ -8,10 +8,6 @@
     <VersionPrefix>1.0.0</VersionPrefix>
```
Comment (dsgouda) [inline src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj]: Please add `<TreatWarningsAsErrors>true</TreatWarningsAsErrors>` here and fix build issues if any. We plan to enforce this soon across all test projects

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks great for the most part

File: src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj (hunk 4 above)
[the remark this answers is not in the document]
Comment (deathly809) [reply src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj]: Done, also did for non-tests.  Let me know if that was wrong.

Comment (deathly809) [reply src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj]: Done, also did for non-tests.  Let me know if that was wrong.

Comment (dsgouda) [reply src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj]: Don't enable this for non-tests yet, there are pieces of generated code that still generate some warnings and we need some fixes there

Comment (deathly809) [reply src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj]: It had no warnings for me.  Should I revert or just leave?

Comment (deathly809) [reply src/AzureStack/Admin/FabricAdmin/Fabric.Admin.Tests/Fabric.Tests.csproj]: Done.

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
Will merge once CIs pass

**Pull request merged**